src/NLog/Targets/EventLogTarget.cs

```diff
@@ -244,20 +242,7 @@ protected override void InitializeTarget()
         {
             base.InitializeTarget();
 
-            var fixedSource = GetFixedSource();
-
-            if (string.IsNullOrEmpty(fixedSource))
-            {
-                InternalLogger.Debug("EventLogTarget(Name={0}): Skipping creation of event source because it contains layout renderers", Name);
```

Comment:
why is this warning being removed?

Reply:
Removes duplicate code. The exact same check is performed farther in `CreateEventSourceIfNeeded()`.